tools/kompos/src/tracingData.ts

```diff
@@ -0,0 +1,110 @@
+/* jshint -W097 */
+'use strict';
+
+import {IdMap, MessageHistoryMessage, MessageData, FarRefData} from './messages';
+import {dbgLog} from './source';
+
+var horizontalDistance = 100,
+    verticalDistance = 100;
+
+export class dataStore {
```

Comment:
Please capitalize class names as `DataStore` to be consistent with the other code in the repo.
I am also wondering whether `DataStore` is not a bit too generic. Perhaps something like `HistoryData` could be more easily understood?